kinto-remote-settings/src/kinto_remote_settings/signer/listeners.py

```diff
@@ -320,7 +320,7 @@ def check_collection_status(
                 raise_invalid(message="Collection already signed")
 
             # Only allow to-sign from to-review if reviewer and no-editor
-            if reviewers_group_uri not in user_principals:
+            if reviewers_group_uri not in user_principals and _to_review_enabled:
```

Comment:
Similar comment as above -- I noticed all of the guard clauses in this branch contain `and _to_review_enabled`. Would it make sense to make `if _to_review_enabled` its own guard with a message stating that review is disabled?

Reply:
Done!